ote/src/cljs/ote/views/main.cljs

```diff
@@ -95,7 +98,24 @@
                                   (e! (fp-controller/->GoToUrl "/user/_logout")))} ]
 
     [ui/menu-item {:style {:color "#FFFFFF"}
-                   :primary-text (r/as-element [language-selection e!])}]])
+                   :primary-text (r/as-element [language-selection e!])}]]))
+
+
+(defn is-page-active
+  "Return true if given current-page belongs to given page-group"
+  [page-group current-page]
+  (cond
+    (= page-group :own-services)
```

Comment:
easier to test if current-page is in a set of pages, than have multiple comparisons.
`(#{:own-services :transport-service :new-service :edit-service} current-page)`

Reply:
Sorry, but I don't understand what I need to change.